### PR TITLE
[Enhance] Update docker file and install.md

### PR DESCRIPTION
**Pull request by dreamerlin**
This PR updates docker file to support PyTorch 1.6 and `install.md`.

### CODE REVIEW

**Review by innerlee — APPROVED** [review]

**Review by innerlee — APPROVED** [review]

**Comment by innerlee** [comment]
good to go after rebase

File: docker/Dockerfile [hunk 1]
```diff
@@ -8,14 +8,18 @@ ENV TORCH_CUDA_ARCH_LIST="6.0 6.1 7.0+PTX"
...
 ENV CMAKE_PREFIX_PATH="$(dirname $(which conda))/../"
 
-RUN apt-get update && apt-get install -y git ninja-build libglib2.0-0 libsm6 libxrender-dev libxext6 \
+RUN apt-get update && apt-get install -y git ninja-build libglib2.0-0 libsm6 libxrender-dev libxext6 ffmpeg \
     && apt-get clean \
     && rm -rf /var/lib/apt/lists/*
 
-# Install mmaction2
+# Install MMCV
+RUN pip install mmcv-full==latest+torch1.6.0+cu101 -f https://openmmlab.oss-accelerate.aliyuncs.com/mmcv/dist/index.html
```
Comment (hellock) [inline docker/Dockerfile:16]: https://download.openmmlab.com

File: docs/install.md [hunk 2]
```diff
@@ -62,14 +62,40 @@ conda install pytorch=1.3.1 cudatoolkit=9.2 torchvision=0.4.2 -c pytorch
...
+pip install mmcv-full==latest+torch1.5.0+cu101 -f http://download.openmmlab.com/mmcv/dist/index.html
+```
+
+See [here](https://github.com/open-mmlab/mmcv#install-with-pip) for different versions of MMCV compatible to different PyTorch and CUDA versions.
+Optionally you can choose to compile mmcv from source by the following command
+
+```shell script
+git clone https://github.com/open-mmlab/mmcv.git
+cd mmcv
+pip install -e .  # package mmcv, which contains no cuda ops, will be installed after this step
```
Comment (hellock) [inline docs/install.md:77]: mmcv-full is preferred

**Comment by innerlee** [comment]
need to rebase

**Pull request merged**